conf/application.conf

```diff
@@ -24,6 +24,7 @@ play.http.errorHandler = "handlers.ErrorHandler"
 # ~~~~
 # Additional play modules can be added here
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
```

Comment:
Do we still need the original `HttpClientModule` enabled if the V2 module has been introduced with the bootstrap upgrade?